Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/configuration.py

```diff
@@ -0,0 +1,562 @@
+config = {
+    "version": 1,
+    "controllers": {
+        "con1": {
+            "type": "opx1",
+            "analog_outputs": {
+                1: {"offset": 0.0, "delay": 0},
+                2: {"offset": 0.0, "delay": 0},
+                7: {
+                    "offset": 0.20529425782807967,
+                    "delay": 0,
+                    "crosstalk": {7: 0.9997004946118614, 8: -0.016448562428420136},
+                    "filter": {"feedforward": [1.0], "feedback": [0.0]},
```

Comment:
I will be careful with these values that are very specific to the KIT set-up... Maybe remove them?

Reply:
I think it's ok for a use-case to have device-specific things. But it should be noted in a comment